config/crd/bases/cloudsql.cloud.google.com_authproxyworkloads.yaml

```diff
@@ -15,8 +15,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.3
-  creationTimestamp: null
+    controller-gen.kubebuilder.io/version: v0.12.0
```

Comment:
Should this and the generated changes below be a part of this PR?

Reply:
Outdated. Merge error.